### ISSUE TITLE
Can't tell if things are running

### ISSUE THREAD

**Issue by seanziee** [sirensolutions/sentinl#115]
Hi,

I have kibana 5.0.1 and I'm testing out KAAE. I did the install and changed the kibana.yml file and made a watch following your instructions.

I'm wondering how to track if KAAE is running and what I've read is that it goes into the Kibana logs. I went into the stout kibana log located at var/log/kibana/kibana.stdout (I'm on centos running kibana as a service) and after going into that file, there is nothing happening at all. Is that where the logs for KAAE go or somewhere else?

**Comment by lmangani** [comment]
WARNING: Kibana 5 support is still sketchy and undergoing major review. Basically everything should run in terms of API and nothing in terms of UI. Sentinl emits several logs to Kibana, and creates its own indices where you can track executions and results from standar Kibana widgets until we have a proper interface ready. Hope this helps!

**Comment by seanziee** [comment]
So I am running from the API (I'm used to watcher which is fully from API).

Things aren't running as they should, as expected because things are new... but I can't find where to get the logs from sentinl. Nothing is coming through the stdout of kibana, so does that mean I installed incorrectly? I followed the instructions on https://github.com/sirensolutions/sentinl/blob/5.0/README.md#installation-steps

Where can I find the errorlogs or anything to say if things are even installed correctly or not?

Thanks so much for making this plugin by the way, I hope to start making plugins soon too :)
